vacuum.yaml

```
@@ -552,9 +554,12 @@ automation:
         from: 'Pending'
         to: 'Ready'
     action:
-      - service: notify.vacuumfile
-        data_template: 
+      action:
+        - target:
```

There is an extra action and the target is not correctly indented.

```suggestion
      - target:
```

Also, thanks a lot for this PR. Hope it is merged soon